salesforce-marketing-cloud/3.2/modules/ROOT/pages/salesforce-mktg-connector-reference.adoc

```diff
@@ -0,0 +1,852 @@
+= Salesforce Marketing Cloud Connector 3.2 Reference - Mule 4
+:page-aliases: connectors::salesforce/salesforce-mktg-connector-reference.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Release Notes: xref:release-notes::connector/salesforce-mktg-connector-release-notes-mule-4.adoc[Salesforce Marketing Cloud Connector Release Notes]
+
+== Configurations
+---
+[[Config]]
+=== Default Configuration
+
+==== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+|Name | String | The name for this configuration. Connectors reference the configuration with this name. | | x
+| Connection a| * <<Config_Basic, Basic Authentication>>
+* <<Config_OauthClientCredentials, OAuth Client Credentials>>
+ | The connection types to provide to this configuration. | | x
+| Read timeout a| Number |  Specifies the amount of time, in the unit defined in Service URL, that the consumer waits for a response before it times out. |  0 |
+| Read timeout time unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  A time unit that qualifies the Read timeout. |  SECONDS |
+| Name a| String |  The identifier of this element used to reference it in other components. |  | x
+| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before Mule considers it eligible for expiration. This does not mean that the platform expires the instance at the exact moment that it becomes eligible. Mule purges the instances as appropriate. |  |
```

Comment:
At your discretion (can be done in a later iteration), update to: 
Configures the minimum amount of time that a dynamic configuration instance can remain idle before Mule considers it eligible for expiration.